tests/StdinFileInfoTest.php

```diff
@@ -222,6 +222,6 @@ public function testNoOpMethods(): void
         $fileInfo->setFileClass('foo1');
         $fileInfo->setInfoClass('foo2');
 
-        $this->addToAssertionCount(1);
+        $this->expectNotToPerformAssertions();
```

Comment:
successful test of no-op methods, I see is no other way than to count this as tested than increase of the assert count as these are tested